docs/reference/commandline/context_create.md

```diff
@@ -23,7 +23,7 @@ Create a context
 Docker endpoint config:
 
 NAME                DESCRIPTION
-from-current        Copy current Docker endpoint configuration
+from                Copy Docker endpoint configuration from an existing context
```

Comment:
Do we still want the `from` for `docker` and `kubernetes`, now that we have the top-level `--from`? Basically; wondering if we want `context create` and `context update` to allow mish-mashing two unrelated contexts.

I can _slightly_ see this being something when _creating_ a new context, but not entirely sure for _updating_ an existing one.

Reply:
I think it is not about "usefulness" but more about symetry: endpoint configuration options would be easier to learn if they are the same for both create and update.